Tests/Validator/UniqueUrlValidatorTest.php

```diff
@@ -99,6 +102,7 @@ public function testValidateWithRootUrlAndNoParent()
 
     public function testValidateWithPageDynamic()
     {
+        $this->skipInPHP55();
```

Comment:
Not sure, if this is a good idea. This code would work on PHP 5.5. 

IMHO this is a dependency problem